manifests/pipecd/templates/deployment.yaml

```diff
@@ -87,6 +87,15 @@ spec:
       {{- if or .Values.serviceAccount.create .Values.serviceAccount.name }}
       serviceAccountName: {{ include "pipecd.serviceAccountName" . }}
       {{- end }}
+      {{- if .Values.quickstart.enabled }}
+      initContainers:
+        - name: minio-checker
+          image: alpine/curl:3.14
+          command: ["sh", "-c", "until curl {{ include "pipecd.fullname" . }}-minio:{{ .Values.minio.port }}; do echo waiting for minio; sleep 2; done"]
+        - name: mysql-checker
+          image: alpine:3.14
+          command: ["sh", "-c", "until nc -z {{ include "pipecd.fullname" . }}-mysql {{ .Values.mysql.port }}; do echo waiting for mysql; sleep 2; done"]
+      {{- end }}
```

Comment:
What a nice idea. 💯 
Just one thing I want to discuss is how about merging those scripts into a single container.
A `dep-waiter` container to ensure that its dependencies (Minio and MySQL) have already been started up.

Just want to reduce the number of containers but not a big deal. So it is up to you.

Reply:
> A `dep-waiter` container to ensure that its dependencies (Minio and MySQL) have already been started up.

I think this idea makes `initContainer` definition more concise too!
It looks like it could be done with a simple change, so I will fix it asap.

Reply:
done:+1:
e87ebd6124e5c53dbe560d2a58a1a150944ae5d1

result:
```
$ kubectl get po --watch
NAME         READY   STATUS    RESTARTS   AGE
alpine-pod   1/1     Running   0          156m
pipecd-ops-87748dfc7-2xzgg   0/1     Pending   0          0s
pipecd-gateway-85658b4584-l4gct   0/1     Pending   0          0s
pipecd-mysql-84cbf4b587-b8lcz     0/1     Pending   0          0s
pipecd-cache-6cc858bc44-gz746     0/1     Pending   0          0s
pipecd-minio-7dd5b8b987-rscx7     0/1     Pending   0          0s
pipecd-minio-7dd5b8b987-rscx7     0/1     Pending   0          0s
pipecd-ops-87748dfc7-2xzgg        0/1     Pending   0          0s
pipecd-mysql-84cbf4b587-b8lcz     0/1     Pending   0          0s
pipecd-cache-6cc858bc44-gz746     0/1     Pending   0          0s
pipecd-gateway-85658b4584-l4gct   0/1     Pending   0          0s
pipecd-server-6bcfc4589b-wg5nd    0/1     Pending   0          0s
pipecd-minio-7dd5b8b987-rscx7     0/1     ContainerCreating   0          0s
pipecd-server-6bcfc4589b-wg5nd    0/1     Pending             0          0s
pipecd-ops-87748dfc7-2xzgg        0/1     Init:0/1            0          0s
pipecd-mysql-84cbf4b587-b8lcz     0/1     ContainerCreating   0          0s
pipecd-cache-6cc858bc44-gz746     0/1     ContainerCreating   0          0s
pipecd-gateway-85658b4584-l4gct   0/1     ContainerCreating   0          2s
pipecd-server-6bcfc4589b-wg5nd    0/1     Init:0/1            0          2s
pipecd-ops-87748dfc7-2xzgg        0/1     Init:0/1            0          3s
pipecd-cache-6cc858bc44-gz746     1/1     Running             0          3s
pipecd-minio-7dd5b8b987-rscx7     1/1     Running             0          3s
pipecd-server-6bcfc4589b-wg5nd    0/1     Init:0/1            0          4s
pipecd-mysql-84cbf4b587-b8lcz     1/1     Running             0          4s
pipecd-gateway-85658b4584-l4gct   0/1     Running             0          5s
pipecd-server-6bcfc4589b-wg5nd    0/1     PodInitializing     0          13s
pipecd-ops-87748dfc7-2xzgg        0/1     PodInitializing     0          13s
pipecd-server-6bcfc4589b-wg5nd    0/1     Running             0          14s
pipecd-server-6bcfc4589b-wg5nd    1/1     Running             0          14s
pipecd-ops-87748dfc7-2xzgg        0/1     Running             0          14s
pipecd-ops-87748dfc7-2xzgg        1/1     Running             0          14s
pipecd-gateway-85658b4584-l4gct   1/1     Running             0          20s
```